homeassistant/components/overkiz/coordinator.py

```diff
@@ -67,10 +69,14 @@ async def _async_update_data(self) -> dict[str, Device]:
             events = await self.client.fetch_events()
         except BadCredentialsException as exception:
             raise ConfigEntryAuthFailed("Invalid authentication.") from exception
+        except TooManyConcurrentRequestsException as exception:
+            raise UpdateFailed("Too many concurrent requests.") from exception
         except TooManyRequestsException as exception:
             raise UpdateFailed("Too many requests, try again later.") from exception
         except MaintenanceException as exception:
             raise UpdateFailed("Server is down for maintenance.") from exception
+        except InvalidEventListenerIdException as exception:
+            raise UpdateFailed(exception) from exception
```

Comment:
Can these new exceptions happen at login as well?

```
    try:
        await client.login()

        setup, scenarios = await asyncio.gather(
            *[
                client.get_setup(),
                client.get_scenarios(),
            ]
        )
    except BadCredentialsException as exception:
        raise ConfigEntryAuthFailed("Invalid authentication") from exception
    except TooManyRequestsException as exception:
        raise ConfigEntryNotReady("Too many requests, try again later") from exception
    except (TimeoutError, ClientError, ServerDisconnectedError) as exception:
        raise ConfigEntryNotReady("Failed to connect") from exception
    except MaintenanceException as exception:
        raise ConfigEntryNotReady("Server is down for maintenance") from exception
```

Reply:
According to my information, no. The `InvalidEventListenerIdException` is only possible on the `fetch_events` call.

The `TooManyConcurrentRequestsException` is a bit vague, however I understood that this only happens on a call after login, when the token just has been refreshed. For now, we haven't seen this behavior during our command on login, only in the coordinator.